aries_cloudagent/protocols/out_of_band/v1_0/messages/invitation.py

```diff
@@ -91,7 +91,11 @@ def to_url(self, base_url: str = None) -> str:
         """
         c_json = self.to_json()
         c_i = bytes_to_b64(c_json.encode("ascii"), urlsafe=True)
-        result = urljoin(base_url or "", "?c_i={}".format(c_i))
+        result = urljoin(
+            base_url
+            or (self.service_blocks[0].service_endpoint if self.service_blocks else ""),
+            "?c_i={}".format(c_i),
```

Comment:
In the OutofBand RFC it's using `?oob=` instead of `?c_i=`, I think everything else is consistent though